test/e2e/scale.go

```diff
@@ -83,11 +87,19 @@ type scaleSpecInput struct {
 	// Can be overridden by variable CAPI_SCALE_CONTROLPLANE_MACHINE_COUNT.
 	ControlPlaneMachineCount *int64
 
-	// WorkerMachineCount defines number of worker machines to be added to each workload cluster.
+	// WorkerMachineCount defines number of worker machines per machine deployment of the workload cluster.
 	// If not specified, 1 will be used.
 	// Can be overridden by variable CAPI_SCALE_WORKER_MACHINE_COUNT.
 	WorkerMachineCount *int64
 
+	// MachineDeploymentCount defines the number of MachineDeployments to be used per workload cluster.
+	// If not specified, 1 will be used.
```

Comment:
Might be worth explaining how this work (it creates clones of the first machine deployment, so the resulting number of nodes will be CAPI_SCALE_WORKER_MACHINE_COUNT * CAPI_SCALE_MACHINE_DEPLOYMENT_COUNT